website/content/en/docs/cli/operator-sdk_alpha.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "operator-sdk alpha"
+---
+## operator-sdk alpha
```

Comment:
Did you add one of the docs manually? It is duplicated. 

<img width="262" alt="Screenshot 2020-05-22 at 17 51 45" src="https://user-images.githubusercontent.com/7708031/82690880-eb794280-9c54-11ea-906f-f2ba180d54d6.png">